network.go

```diff
@@ -67,6 +67,11 @@ func localIPv4MCastAddrs() ([]string, error) {
 				// Not IPv4.
 				continue
 			}
+			// Fix #33: consider NOT loopback IP addresses
+			if addr.IP.IsLoopback() {
```

Comment:
Could you test if checking discarding interfaces for which `iface.Flags&net.FlagLoopback != 0`, a few lines earlier?

Reply:
(This would also move the check earlier in this function)

Reply:
Yep, did work. Thanks for hint, is more effective.